features/pages/flaw_detail_page.py

```diff
@@ -241,6 +254,14 @@ def set_select_value(self, field):
             updated_value = current_value
         return updated_value
 
+    def click_btn(self, btn_element):
+        element = getattr(self, btn_element)
+        element.click_button()
+
+    def wait_msg(self, msg_element):
```

Comment:
No need to add this function, please check `BasePage` class, i extract common function to this class and all Page class inherit it.

Reply:
Thanks. Removed.